docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,223 @@
+# **Feature Engineering for W&D (Wide and Deep Learning) using Friesian**
```

Comment:
1.	Key Concepts
2.	W&D implementation on Friesian
2.1	init_orca_context
2.2	Load data into FeatureTable (briefly explain `read_csv` and `cache` here)
2.3	Data processing using FeatureTable (briefly explain `category_encode`, ` add_negative_samples`, `cross_columns` here)
2.4	Define W&D model
2.5	Distributed Training using Orca Estimator